#request
uint8 payload_index
uint8 exposure_mode
uint8 exposure_compensation
---
#reponse
bool result

